### PR TITLE
koji_promote: log upload progress

### PR DESCRIPTION
**Pull request by twaugh**

### CODE REVIEW

**Comment by lcarva** [comment]
LGTM

File: atomic_reactor/plugins/exit_koji_promote.py [hunk 1]
```diff
@@ -103,6 +123,7 @@ def __init__(self, tasker, workflow, kojihub, url,
                                   namespace=self.namespace)
         self.osbs = OSBS(osbs_conf, osbs_conf)
         self.build_id = None
+        self.nvr_image = None
```
Comment (vrutkovs) [inline atomic_reactor/plugins/exit_koji_promote.py]: Nitpick: this probably comes from a different commit?

**Comment by vrutkovs** [comment]
A small nitpick, otherwise looks good

File: atomic_reactor/plugins/exit_koji_promote.py (hunk 1 above)
Comment (twaugh) [inline atomic_reactor/plugins/exit_koji_promote.py]: I bundled three fixes for warnings together in one commit. Only one of the warnings is new with this PR.

**Pull request merged**